### ISSUE TITLE
SoX 14.4.2-18: cannot link executable

### ISSUE THREAD

**Issue by giuliolunati** [termux/termux-packages#6140]
**Problem description**

running sox I get error:
CANNOT LINK EXECUTABLE "sox": library "libomp.so" not found

**Steps to reproduce**

launch sox from prompt
**Expected behavior**

**Additional information**

```
Packages CPU architecture:
arm
Subscribed repositories:
# sources.list
deb https://main.termux-mirror.ml/ stable main
# sources.list.d/pointless.list
deb https://its-pointless.github.io/files/24 termux extras
# game-repo (sources.list.d/game.list)
deb https://dl.bintray.com/grimler/game-packages-24 games stable
# unstable-repo (sources.list.d/unstable.list)
deb https://dl.bintray.com/xeffyr/unstable-packages unstable main
# root-repo (sources.list.d/root.list)
deb https://dl.bintray.com/grimler/termux-root-packages-24 root stable
# science-repo (sources.list.d/science.list)
deb https://dl.bintray.com/grimler/science-packages-24 science stable
# x11-repo (sources.list.d/x11.list)
deb https://dl.bintray.com/xeffyr/x11-packages x11 main
Updatable packages:
All packages up to date
Android version:
10
Kernel build information:
Linux localhost 3.4.113-lineageos-g36dddde #1 SMP PREEMPT Sun Nov 22 03:14:30 UTC 2020 armv7l Android
Device manufacturer:
samsung
Device model:
SM-G900F

```

**Comment by xeffyr** [comment]
Should be fixed in `sox v14.4.2-19`.

**Comment by giuliolunati** [comment]
Yes, it works. Thank you!
